example-viewbinding/build.gradle

```diff
@@ -0,0 +1,41 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion rootProject.sdkVersion
+
+    defaultConfig {
+        applicationId "com.xwray.groupie.example.viewbinding"
+        minSdkVersion rootProject.minimumSdkVersion
+        targetSdkVersion rootProject.sdkVersion
+        versionCode 1
+        versionName "1.0"
+        vectorDrawables.useSupportLibrary true
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+
+    dataBinding {
+        enabled = true
+    }
+
+    viewBinding {
+        enabled = true
+    }
+
+    lintOptions {
+        abortOnError false
+    }
+}
+
+dependencies {
+    implementation project(':library-viewbinding')
+    implementation project(':example-shared')
+    implementation "androidx.appcompat:appcompat:1.0.2"
```

Comment:
for this to be a more up to date example, please use all latest library versions here

Reply:
updated ✅(98caac2)